Remove rake

timcraft opened the pull request:
Removes the Rakefile and the rake gem dependency.

Rake doesn't appear to be used for anything—there aren't any rake tasks defined, and the travis validation script is using bash/node.

fabianrbz commented:
Hi @timcraft, we do have some tasks defined under `lib/tasks/` and we also use Rails' ones (you can take a look at them by running `rake -T`) so we can't remove this dependency

tbedford commented:
@fabianrbz - I think there are conflicts here that need resolving? Is that right that Rake is not used? I thought I saw in a comment it is used? Thanks.

fabianrbz commented:
@tbedford we need to fix the conflicts. Yeah, I edited the comment but somehow i ended up deleting it 🤦‍♂ . We don't use it in this repo.

timcraft commented:
@tbedford @fabianrbz I've rebased and fixed up the conflict in the Gemfile.

Confusing I know but `lib/tasks` doesn't exist in the repo, and `rake -T` displays nothing.

tbedford commented:
Thanks Tim!
